### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by AshParker19**

### CODE REVIEW

**Review by rajh-phuyal — COMMENTED** [review]
Ok

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ENV PYTHONUNBUFFERED True
+
+ENV APP_HOME /back-end
+WORKDIR $APP_HOME
+COPY . ./
+
+RUN pip install --no-cache-dir --upgrade pip
+RUN pip install --no-cache-dir -r requirements.txt
+
+CMD exec gunicorn --bind :$PORT --workers 1 --threads 8 --timeout 0 app:app
```
[the remark this answers is not in the document]
Comment (AshParker19) [reply Dockerfile:12]: I'm not quite sure how it it's gonna affect the performance. Need to experiment about it more.

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
-# Google-Hackathon-June-2024
\ No newline at end of file
+# Google-Hackathon-June-2024
+
+---
+## How to deploy the project on Google Cloud:
+1. Install Google Cloud CLI for your OS: https://cloud.google.com/sdk/docs/install
+    (You can check if it's installed by running `gcloud --version` in the terminal.)
+2. Update your environment with requirements from the `requirements.txt` file.
```
[the remark this answers is not in the document]
Comment (AshParker19) [reply README.md]: I meant install updated requirements

File: config.py [hunk 3]
```diff
@@ -1,11 +1,13 @@
 from flask_sqlalchemy import SQLAlchemy
 from flask import Flask
 from dotenv import load_dotenv
+from flask_cors import CORS
 import os
 
 load_dotenv()
 
 app = Flask(__name__)
+CORS(app)
```
[the remark this answers is not in the document]
Comment (AshParker19) [reply config.py:10]: Thanks!

**Review by rajh-phuyal — APPROVED** [review]
Good work!

**Review by oks-erm — APPROVED** [review]
noice

**Pull request merged**